frontend/src/util/helper.js

```diff
@@ -212,7 +212,7 @@ export const toggleFavicon = (hasNewNotification) => {
       )
       // FIXME - GM - 2020/08/18 - Replace this hardcoded values to webpack variables
       // https://github.com/tracim/tracim/issues/2098
-      context.fillStyle = '#3F9FF7'
+      context.fillStyle = hasUnreadMention ? '#dc3545' : '#3F9FF7'
```

Comment:
I think you can use `#ff8a00` to be coherent with #4858 

```suggestion
      context.fillStyle = hasUnreadMention ? '#ff8a00' : '#3F9FF7'
```